doc/source/whatsnew/v1.4.0.rst

```diff
@@ -355,7 +355,7 @@ Groupby/resample/rolling
 - Bug in :meth:`pandas.DataFrame.ewm`, where non-float64 dtypes were silently failing (:issue:`42452`)
 - Bug in :meth:`pandas.DataFrame.rolling` operation along rows (``axis=1``) incorrectly omits columns containing ``float16`` and ``float32`` (:issue:`41779`)
 - Bug in :meth:`Resampler.aggregate` did not allow the use of Named Aggregation (:issue:`32803`)
--
+- Bug in :class:`GroupbyIndexer` and :class:`FixedForwardWindowIndexer` leading to segfaults and incorrect windows (:issue:`43267`)
```

Comment:
* ``` :class:`pandas.api.indexers.FixedForwardWindowIndexer ```

* instead of mentioning `GroupbyIndexer`, mention the `groupby.rolling` operation
* Could you be more specific with "incorrect windows"

Reply:
@mroeschke would these changes be okay for 1.3.3? https://github.com/pandas-dev/pandas/issues/43267#issuecomment-908245101 or would we want to split this PR?

Reply:
Sure this would be okay for 1.3.3. @dsm054 could you moved this note to 1.3.3?